src/mmw/apps/modeling/mapshed.py

```diff
@@ -146,8 +149,8 @@ def animal_energy_units(geom):
         values = cursor.fetchone()  # Only one row since aggregate query
         aeu = dict(zip(columns, values))
 
-        livestock_aeu = round(sum(aeu[animal] for animal in LIVESTOCK))
-        poultry_aeu = round(sum(aeu[animal] for animal in POULTRY))
+        livestock_aeu = round(sum(aeu[animal] for animal in LIVESTOCK)) or 0
+        poultry_aeu = round(sum(aeu[animal] for animal in POULTRY)) or 0
```

Comment:
Actually, the constants `LIVESTOCK` and `POULTRY` are composed of the column names returned from this query. They should **always** match. If they don't, then we _should_ raise an exception.

However, if the value of one of those fields is `None`, that could interfere with the `sum`. I think moving the `0` inside, like this:

```python
livestock_aeu = round(sum(aeu[animal] or 0 for animal in LIVESTOCK))
```

might be better, so I will switch to that.